components/infobox/wikis/rainbowsix/infobox_league_custom.lua

```diff
@@ -146,59 +133,57 @@ function CustomLeague:_standardiseRawDate(dateString)
 	return dateString
 end
 
-function CustomLeague:defineCustomPageVariables(args)
-	-- Variables with different handling compared to commons
-	Variables.varDefine('tournament_liquipediatiertype',
-		Variables.varDefault('tournament_liquipediatiertype', DEFAULT_TIERTYPE))
-
-	if CustomLeague:_validPublisherTier(args.ubisofttier) then
-		Variables.varDefine('tournament_publishertier', args.ubisofttier:lower())
-	end
+---@param args table
+function CustomLeague:customParseArguments(args)
+	self.data.liquipediatiertype = self.data.liquipediatiertype or DEFAULT_TIERTYPE
+	self.data.publishertier = self:_validPublisherTier(args.ubisofttier) and args.ubisofttier:lower()
+		or self.data.publishertier
+end
 
+---@param args table
+function CustomLeague:defineCustomPageVariables(args)
 	--Legacy vars
 	Variables.varDefine('tournament_ticker_name', args.tickername or '')
 	Variables.varDefine('tournament_tier', args.liquipediatier or '')
-	Variables.varDefine('tournament_tier_type', Variables.varDefault('tournament_liquipediatiertype'))
+	Variables.varDefine('tournament_tier_type', self.data.liquipediatiertype)
 	Variables.varDefine('tournament_prizepool', args.prizepool or '')
-	Variables.varDefine('tournament_mode', args.mode or '')
 
 	--Legacy date vars
-	local sdate = Variables.varDefault('tournament_startdate', '')
-	local edate = Variables.varDefault('tournament_enddate', '')
-	Variables.varDefine('tournament_sdate', sdate)
-	Variables.varDefine('tournament_edate', edate)
-	Variables.varDefine('tournament_date', edate)
-	Variables.varDefine('date', edate)
-	Variables.varDefine('sdate', sdate)
-	Variables.varDefine('edate', edate)
+	Variables.varDefine('tournament_sdate', self.data.startDate)
+	Variables.varDefine('tournament_edate', self.data.endDate)
+	Variables.varDefine('tournament_date', self.data.endDate)
+	Variables.varDefine('date', self.data.endDate)
+	Variables.varDefine('sdate', self.data.startDate)
+	Variables.varDefine('edate', self.data.endDate)
 end
 
+---@param args table
+---@return string[]
 function CustomLeague:getWikiCategories(args)
-	local categories = {}
-
-	if not Game.name{game = _args.game} then
-		table.insert(categories, 'Tournaments without game version')
-	else
-		table.insert(categories, Game.name{game = _args.game} .. ' Competitions')
-	end
-
-	if CustomLeague:_platformLookup(args.platform) then
-		table.insert(categories, CustomLeague:_platformLookup(args.platform) .. ' Tournaments')
-	end
+	local game = Game.name{game = args.game}
+	local platform = self:_platformLookup(args.platform)
 
-	return categories
+	return {
+		game and (game .. 'Competitions') or 'Tournaments without game version',
+		platform and (platform .. ' Tournaments') or nil,
+	}
```

Comment:
Currently breaks the categories. ``Siege Competitions`` ``Siege Tournaments`` (the old categories) are replaced with ``Tom Clancy's Rainbow Six Siege Competitions`` and ``Tom Clancy's Rainbow Six Vegas 2 Competitions``. 

I'm not totally opposed to this change, but the new category names are awfully long. 
If possible, cutting the ``Tom Clancy's`` from the categories would be preferred

Reply:
don't think it is due to this PR, think it is https://github.com/Liquipedia/Lua-Modules/pull/3655 causing it
could you check if it actually changes between dev and non dev?

Reply:
it almost certainly is

Reply:
the old code literally used the same values